docs/dev/connectors/streamfile_sink.md

```diff
@@ -381,4 +453,6 @@ aggressively and take a savepoint with some part-files being not fully uploaded,
 before the job is restarted. This will result in your job not being able to restore from that savepoint as the
 pending part-files are no longer there and Flink will fail with an exception as it tries to fetch them and fails.
 
+<!-- SAVEPOINTS -->
```

Comment:
?

Reply:
A forgotten "note to self" about what to include. Removed.